sentinelhub/constants.py

```diff
@@ -409,31 +406,12 @@ def get_string(self):
             return 'application/x-tar'
         if self is MimeType.JSON:
             return 'application/json'
-        if self in [MimeType.TIFF_d8, MimeType.TIFF_d16, MimeType.TIFF_d32f]:
-            return 'image/{}'.format(self.value)
         if self is MimeType.JP2:
             return 'image/jpeg2000'
         if self is MimeType.RAW:
             return self.value
         return mimetypes.types_map['.' + self.value]
 
-    def get_sample_type(self):
-        """ Returns sampleType used in Sentinel-Hub evalscripts.
-
-        :return: sampleType
-        :rtype: str
-        :raises: ValueError
-        """
-        try:
-            return {
-                MimeType.TIFF: 'INT16',
-                MimeType.TIFF_d8: 'INT8',
-                MimeType.TIFF_d16: 'INT16',
-                MimeType.TIFF_d32f: 'FLOAT32'
-            }[self]
-        except KeyError as exception:
-            raise ValueError('Type {} is not supported by this method'.format(self)) from exception
-
     def get_expected_max_value(self):
         """ Returns max value of image `MimeType` format and raises an error if it is not an image format
 
```

Comment:
A line below this one has a comment about `MimeType.TIFF_d32f` that should be removed

Reply:
Removed the comment.

A side note: I believe the whole `get_expected_max_value` is (to some extent) wrong anyway, as the expected maximum value is actually based on the requested `SampleType`.